query_base.go

```diff
@@ -146,6 +149,32 @@ func (q *baseQuery) GetTableName() string {
 	return ""
 }
 
+func (q *baseQuery) setComment(c string) {
+	q.comment = c
+}
+
+func (q *baseQuery) appendComment(b []byte) []byte {
```

Comment:
Could you please rework this to accept `ctx context.Context` and then extract the comment from the context in this function?

Then we can remove `commentFromContext`.

Reply:
Thanks for the review @vmihailenco! Updated as you requested and rebased on top of master.